src/main/java/io/jenkins/plugins/audit/listeners/NodeListener.java

```diff
@@ -0,0 +1,68 @@
+package io.jenkins.plugins.audit.listeners;
+
+import hudson.Extension;
+import hudson.ExtensionList;
+import hudson.model.Node;
+import io.jenkins.plugins.audit.event.CreateNode;
+import io.jenkins.plugins.audit.event.DeleteNode;
+import io.jenkins.plugins.audit.event.Logout;
+import org.apache.logging.log4j.audit.LogEventFactory;
+
+import javax.annotation.Nonnull;
+import javax.swing.undo.CannotRedoException;
```

Comment:
Not sure how this import ended up here. Can you optimize imports by removing unused imports?

Reply:
I am also not sure. Maybe a result of a lot of copy-pastes to understand the codebase. I removed it.